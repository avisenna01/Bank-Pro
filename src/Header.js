import React from "react";
import { makeStyles } from "@material-ui/core/styles";
import Paper from "@material-ui/core/Paper";

const useStyles = makeStyles(theme => ({
  root: {
    padding: theme.spacing(3, 9),
    backgroundColor: "#0a0a0a",
    color: "white",
    borderRadius: "0px"
  }
}));

const Header = props => {
  const classes = useStyles();

  return <Paper className={classes.root}>LOGO</Paper>;
};

export default Header;
