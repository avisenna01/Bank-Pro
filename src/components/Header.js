import React from "react";
import { makeStyles } from "@material-ui/core/styles";
import Paper from "@material-ui/core/Paper";
import logo from "../assets/apple1.png"

const useStyles = makeStyles(theme => ({
  root: {
    padding: theme.spacing(2, 9),
    backgroundColor: "#0a0a0a",
    color: "white",
    borderRadius: "0px"
  }
}));

const Header = () => {
  const classes = useStyles();

  return <Paper className={classes.root}><img alt="logo" src={logo} width="20px" height="24px" /></Paper>;
};

export default Header;
