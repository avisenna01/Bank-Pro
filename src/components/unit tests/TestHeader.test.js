import React from 'react'
import ReactDOM from 'react-dom';
import Header from "../Header";
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

const rendercomponent = () => (shallow(<Header />));

describe('Header component try 1', () => {
  it('snapshot renders', () => {
    const renderedcomponent = rendercomponent();
    expect(renderedcomponent.find('div').length).toEqual(0);
  });
});

describe("Header component try 2", () => {
  test("snapshot renders", () => {
    const button = renderer.create(<Header />);
    expect(button.toJSON()).toMatchSnapshot();
  });
});

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Header />, div);
});