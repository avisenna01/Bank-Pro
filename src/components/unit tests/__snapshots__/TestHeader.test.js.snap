// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component try 2 snapshot renders 1`] = `
<div
  className="MuiPaper-root MuiPaper-elevation1 makeStyles-root-1 MuiPaper-rounded"
>
  <img
    alt="logo"
    height="24px"
    src="apple1.png"
    width="20px"
  />
</div>
`;
